### PR TITLE
DO-15: Completed the List view for mobile.

### PR DESCRIPTION
**Pull request by ayushmalik2110**

### CODE REVIEW

File: DO/src/components/categories/member/MembersListView.jsx [hunk 1]
```diff
@@ -114,17 +117,17 @@ class MembersListView extends React.Component {
       );
     }
     return (
-      <div className='memberListView'>
-        {this.renderMemberListViewHeader()}
+      <div className='member-list-view-container'>
+        {/*{this.renderMemberListViewHeader()}*/}
```
Comment (mohitsaladiya) [inline DO/src/components/categories/member/MembersListView.jsx]: Commented out code?

File: DO/src/components/categories/member/MembersListView.jsx [hunk 2]
```diff
@@ -136,15 +139,15 @@ class MembersListView extends React.Component {
   render() {
     return(
       <>
-        <div className={'member-filter'}>
+        {/*<div className={'member-filter'}>
```
Comment (mohitsaladiya) [inline DO/src/components/categories/member/MembersListView.jsx]: Here as well

File: DO/src/components/categories/member/MembersListView.jsx (hunk 1 above)
Comment (ayushmalik2110) [reply DO/src/components/categories/member/MembersListView.jsx]: This commented out code is actually a implementation of the header. Currently we are not showing it. Will have to display it or remove it as per the final design.

File: DO/src/components/categories/member/MembersListView.jsx (hunk 2 above)
Comment (ayushmalik2110) [reply DO/src/components/categories/member/MembersListView.jsx]: This is the implementation of filter. Will have to update it as well as per the final design.

**Pull request merged**